frontend/src/APIClients/mutations.ts

```diff
@@ -71,5 +71,70 @@ export interface ResetPasswordResponse {
   ok: boolean;
 }
 
+export const SIGNUP = gql`
+  mutation SignUp(
+    $firstName: String!
+    $lastName: String!
+    $email: String!
+    $password: String!
+  ) {
+    signup(
+      firstName: $firstName
+      lastName: $lastName
+      email: $email
+      password: $password
+    ) {
+      id
+      firstName
+      lastName
+      email
+      role
+      accessToken
+      refreshToken
+    }
+  }
+`;
+
 // Stories
-// TODO: move mutation gql strings here
+export const UPDATE_STORY_TRANSLATION_CONTENTS = gql`
+  mutation UpdateStoryTranslationContents(
+    $storyTranslationContents: [StoryTranslationContentRequestDTO]
+  ) {
+    updateStoryTranslationContents(
+      storyTranslationContents: $storyTranslationContents
+    ) {
+      story {
+        id
+      }
+    }
+  }
+`;
+
+export const CREATE_TRANSLATION = gql`
+  mutation CreateStoryTranslation(
+    $storyTranslationData: CreateStoryTranslationRequestDTO!
+  ) {
+    createStoryTranslation(storyTranslationData: $storyTranslationData) {
+      story {
+        id
+      }
+    }
+  }
+`;
+export type CreateTranslation = {
```

Comment:
Is there a good practice around what to name the response objects? Wondering if something like `CreateTranslationResponseDTO` is clearer

Reply:
going with `CreateTranslationResponse` as per patterns in `EntityAPIClient.ts`